GRAPHICS/SetParticleFxLoopedColour.md

```diff
@@ -8,15 +8,15 @@ ns: GRAPHICS
 void SET_PARTICLE_FX_LOOPED_COLOUR(int ptfxHandle, float r, float g, float b, BOOL p4);
 ```
 
-```
-only works on some fx's  
-p4 = 0  
-```
-
 ## Parameters
 * **ptfxHandle**: 
-* **r**: 
-* **g**: 
-* **b**: 
+* **r**: range from 0.0 to 1.0
+* **g**: range from 0.0 to 1.0
+* **b**: range from 0.0 to 1.0
 * **p4**: 
```

Comment:
Missing param is if is local only. False by default.

Reply:
It will also answer @4mmonium comments on
> It would also be nice if you clarified on which fx's this works on and elaborated on why p4 needs to be false, rather than just p4 = 0 🙂